src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/ApplicationTab.java

```diff
@@ -170,9 +167,6 @@ private void addApplicant2(final Tab.TabBuilder<CaseData, UserRole> tabBuilder)
             .label("LabelApplicant2DetailsAreConfidential-Heading",
                 "applicant2ContactDetailsType=\"private\"",
                 "#### ${labelContentTheApplicant2UC}'s contact details are confidential")
-            .field("applicant2PhoneNumber", APPLICANT_2_CONTACT_DETAILS_PUBLIC)
-            .field("applicant2Email", APPLICANT_2_CONTACT_DETAILS_PUBLIC)
-            .field("applicant2Address", APPLICANT_2_CONTACT_DETAILS_PUBLIC)
```

Comment:
Initially I looked to add role-based access to specific fields within the application tab, but it doesn't seem to be supported by CCD config generator (the entire tab is hidden).